Parsers/ASimFileEvent/test/VMwareCarbonBlackCloud_vimFileEvent_SchemaTest.csv

```diff
@@ -0,0 +1,86 @@
+"(2) Info: Missing optional field [EventOriginalSubType]"
+"(2) Info: Missing optional field [EventProductVersion]"
+"(2) Info: Missing optional field [EventReportUrl]"
+"(2) Info: Missing optional field [EventResultDetails]"
+"(2) Info: Missing optional field [EventSubType]"
+"(2) Info: Missing optional field [HttpUserAgent]"
+"(2) Info: Missing optional field [NetworkApplicationProtocol]"
+"(2) Info: Missing optional field [RuleName]"
+"(2) Info: Missing optional field [RuleNumber]"
+"(2) Info: Missing optional field [Rule]"
+"(2) Info: Missing optional field [SrcDescription]"
+"(2) Info: Missing optional field [SrcDeviceType]"
+"(2) Info: Missing optional field [SrcDomain]"
+"(2) Info: Missing optional field [SrcDvcId]"
+"(2) Info: Missing optional field [SrcDvcScopeId]"
+"(2) Info: Missing optional field [SrcDvcScope]"
+"(2) Info: Missing optional field [SrcFQDN]"
+"(2) Info: Missing optional field [SrcFileCreationTime]"
+"(2) Info: Missing optional field [SrcFileDirectory]"
+"(2) Info: Missing optional field [SrcFileExtension]"
+"(2) Info: Missing optional field [SrcFileMD5]"
+"(2) Info: Missing optional field [SrcFileMimeType]"
+"(2) Info: Missing optional field [SrcFileName]"
+"(2) Info: Missing optional field [SrcFileSHA1]"
+"(2) Info: Missing optional field [SrcFileSHA256]"
+"(2) Info: Missing optional field [SrcFileSHA512]"
+"(2) Info: Missing optional field [SrcFileSize]"
+"(2) Info: Missing optional field [SrcGeoCity]"
+"(2) Info: Missing optional field [SrcGeoCountry]"
+"(2) Info: Missing optional field [SrcGeoLatitude]"
+"(2) Info: Missing optional field [SrcGeoLongitude]"
+"(2) Info: Missing optional field [SrcGeoRegion]"
+"(2) Info: Missing optional field [SrcHostname]"
+"(2) Info: Missing optional field [SrcPortNumber]"
+"(2) Info: Missing optional field [Src]"
+"(2) Info: Missing optional field [TargetAppId]"
+"(2) Info: Missing optional field [TargetAppName]"
+"(2) Info: Missing optional field [TargetFileCreationTime]"
+"(2) Info: Missing optional field [TargetFileDirectory]"
+"(2) Info: Missing optional field [TargetFileExtension]"
+"(2) Info: Missing optional field [TargetFileMimeType]"
+"(2) Info: Missing optional field [TargetFileSHA1]"
+"(2) Info: Missing optional field [TargetFileSHA512]"
+"(2) Info: Missing optional field [TargetFileSize]"
+"(2) Info: Missing optional field [TargetUrl]"
+"(2) Info: Missing optional field [ThreatCategory]"
+"(2) Info: Missing optional field [ThreatConfidence]"
+"(2) Info: Missing optional field [ThreatFilePath]"
+"(2) Info: Missing optional field [ThreatFirstReportedTime]"
+"(2) Info: Missing optional field [ThreatId]"
+"(2) Info: Missing optional field [ThreatIsActive]"
+"(2) Info: Missing optional field [ThreatLastReportedTime]"
+"(2) Info: Missing optional field [ThreatName]"
+"(2) Info: Missing optional field [ThreatOriginalConfidence]"
+"(2) Info: Missing optional field [ThreatOriginalRiskLevel]"
+"(2) Info: Missing optional field [ThreatRiskLevel]"
```

Comment:
Please set Threat fields

Reply:
Not Available

Reply:
@jayeshprajapaticrest @niralishah-crest 
the logs you got are file activities logs from user's endpoint or device?
Does Carbon black doesn't support threat events or we couldn't generate them?

Reply:
@vakohl 
Whatever data we have currently in that threat related data for file schema is not available.